src/frontends/common/src/manager.cpp

```diff
@@ -226,7 +226,7 @@ FrontEnd::Ptr FrontEndManager::load_by_framework(const std::string& framework) {
 }
 
 FrontEnd::Ptr FrontEndManager::load_by_model_impl(const std::vector<ov::Any>& variants) {
-    return m_impl->load_by_model(variants);
+    return m_impl->load_by_model(to_wstring_if_needed(variants));
```

Comment:
Why this conversion is required? the Frontends suppor string and wstring if enabled.

This function is not always called as it is virtual and depends on usage context the frontend implementation can be called here or by resolving the VTabele. If 2nd option will be used this conversion is not applied.

There should be unit test for each frontend to call load model with model path which causing issues.

Reply:
The problem occurs only in python code, as python 'str' is always translated to std::string, but it may contain unicode symbols. This problem was already fixed for `Core().read_model()` in this PR: https://github.com/openvinotoolkit/openvino/pull/24606

Here I extend this solution for frontend.load(), frontend_manager.load_by_model() and frontend.supported(), because these methods have python bindings, which are used inside convert_model() and they cause the same problem with unicode paths.

Regarding the tests, the main goal was to test python bindings with unicode paths. But I can extend tests for other frontends if there is a possibility that some of them don't support wstring.

Reply:
Regarding running the loading model by resolving the VTabele. Could you please give an example of loading model this way? If we move the conversion to wstring to [`load_by_model()`](https://github.com/openvinotoolkit/openvino/blob/15257f1bac12afee96b7f53fe0d5d24571d1f319/src/frontends/common/src/manager.cpp#L245) will it fix the problem?

Reply:
Is this issue applicable only to frontend loading? If no, maybe we can move this unicode->wstring conversion to python bindings? It can be also used for other similar issues.

Reply:
Using `load_by_framework` or `load_by_model` from manager should always call `load_by_model_impl` in base class. The issue can be when FE created like:
` auto fe = std::make_shared<ov::frontend::onnx::FrontEnd>();`

If this issue is more related with python and binding will try the @olpipi suggestion to fix it there because Frontends have support to read path as string or wstring.

Reply:
Moved the conversion to wstring to bindings, please take a look.